diesel_derives/src/lib.rs

```diff
@@ -394,10 +394,11 @@ pub fn derive_query_id(input: TokenStream) -> TokenStream {
 ///
 /// This trait can only be derived for structs, not enums.
 ///
-/// **When this trait is derived, it will assume that the order of fields on your
-/// struct match the order of the fields in the query. This means that field
+/// **When this trait is derived, it will assume that __all fields on your struct__
+/// matches __all fields in the query__, including the order. This means that field
```

Comment:
We should maybe be clearer on what `matches` exactly means in this context.

Reply:
I changed it to "is the same as"

Reply:
Sorry for being so pedantic here, but I wouldn't say the "the field are the same as", as this would imply that they are the same fields. The query contains different types then the type deriving `Queryable`.

Reply:
But I thought they need to have all the fields?

Reply:
Used `matches`